MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.cxx

```diff
@@ -159,6 +164,19 @@ void vtkMRMLMarkupsToModelNode::ReadXMLAttributes( const char** atts )
       }
       SetInterpolationType(interpolationType);
     }
+    else if ( ! strcmp( attName, "PointParameterType" ) )
+    {
+      int pointParameterType = 0;
+      std::stringstream nameString;
+      nameString << attValue;
+      nameString >> pointParameterType;
```

Comment:
Don't store magic numbers in files. How would you know what PointParameterType="1" means? You could document it somewhere but then you have to create and maintain that documentation and make it easy to find for people who want to interpret your files, etc.
Instead, convert the enum to a string and store that in file - as it is done for InterpolationType.